app/code/community/Aijko/WidgetImageChooser/controllers/Adminhtml/Cms/Wysiwyg/Images/ChooserController.php

```diff
@@ -32,4 +32,8 @@ public function onInsertAction()
 
         $this->getResponse()->setBody($mediaPath);
     }
+    function _isAllowed()
```

Comment:
Hi @schmengler,

please add a space and doc block to this new function, as well as defining it's public visibility.